test/e2e/kubelet_perf.go

```diff
@@ -184,7 +184,7 @@ var _ = Describe("Kubelet [Serial] [Slow]", func() {
 			})
 		}
 	})
-	Describe("experimental resource usage tracking", func() {
+	Describe("experimental resource usage tracking [Feature:ExperimentalResourceUsageTracking]", func() {
```

Comment:
Ah, I guess this is still an issue for the GKE soak cluster.

Why don't we just skip running this on GKE then, rather than labeling this feature?